recipes/cpc2/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "cpc2" %}
+{% set version = "1.0.1" %}
+{% set file_ext = "tar.gz" %}
+{% set sha256 = "31f3d1e2466aea25f03ffcb5abd4556cd17862ef65ed3a1e42f3d99665529d08" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/gao-lab/CPC2_standalone/archive/refs/tags/v{{ version }}.{{ file_ext }}
+  sha256: {{ sha256 }}
+  path: ..
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - setuptools
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - setuptools 
+    - pip
+    - python
+    - numpy
+  run:
+    - biopython
+    - six
+    - python
```

Looking closer, this does not appear to be a python package (in the sense of having a setup.py), so setuptools and pip are not needed either. You may need `libsvm` in `host:` though.

```suggestion
  run:
    - biopython
    - six
    - python
    - numpy
```

---

I just removed the setuptools and pip from the yaml file. Regarding the libsvm, this cpc2 tool failed to run if the libsvm wasn't compiled in the build.sh script.